BrickKit/app/src/main/java/com/wayfair/brickkitdemo/bricks/UsedBrick.java

```diff
@@ -59,10 +59,12 @@ public void setText(CharSequence text) {
 
     @Override
     public void onBindData(final BrickViewHolder viewHolder) {
-        UsedBrickHolder holder = (UsedBrickHolder) viewHolder;
-        holder.textView.setText(text);
-        if (isEnabled()) {
-            holder.itemView.setOnClickListener(onTouch());
+        if(viewHolder instanceof UsedBrickHolder) {
```

Comment:
This is a fix for a class cast exception.

Reply:
surprised checkstyle didn't flag you...should be a space between "if("

Reply:
I fixed it.